Gopkg.toml

```diff
@@ -88,3 +88,7 @@
 [[constraint]]
   name = "github.com/gobuffalo/packr"
   version = "1.13.5"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/bobappleyard/readline"
```

Comment:
I'm concerned about this library's dependency on `readline`.  Couple simpler alternatives to consider?

* https://github.com/tcnksm/go-input
* https://github.com/chzyer/readline

Reply:
I'm not tied to this package at all. Readline looks good, I'll swap it in instead